jekyll/_cci2/customizing-ios.md

```diff
@@ -0,0 +1,429 @@
+
+## Getting Started
+
+Select a macOS project you would like to build on the [Add
+Projects page](https://circleci.com/add-projects). **Note:** If your project does not appear on the OS X tab, it may be listed as a Linux project. Add your project as a Linux build, then change
+it to a macOS build by enabling the Build OS X project option on the
+Project Settings > Build Environment page.
+
+## Basic setup
+
+After enabling macOS builds for your project, share
+the scheme that is going to be built on CircleCI so that we run the
+correct build actions. Here is how to share an existing scheme in Xcode:
+
+1. Choose Product > Scheme > Manage Schemes.
+2. Select the Shared option for the scheme to share, and click Close.
+3. Choose Source Control > Commit.
+4. Select the Shared Data folder.
+5. Enter your commit message in the text field.
+6. Select the "Push to remote" option (if your project is managed with Git).
+7. Click the Commit Files button.
+A new `.xcscheme` file is located in the
+`xcshareddata/xcschemes` folder under your Xcode project. 
+8. Commit this file to your git repository so that CircleCI can access it.
+
+Simple projects should run with minimal or no configuration.
+
+By default, CircleCI will:
+
+* **Build your project with Xcode 7.0 on OSX 10.11** - You can
+select a specific version of Xcode by specifying the major and minor version in
+a [config.yml file]( {{ site.baseurl }}/2.0/configuration-reference/) in the `.circleci` directory of your
+repository. 
+* **Install any Ruby gems specified in a Gemfile** - We will run `bundle
+  install` and cache the installed gems.
+* **Install any dependencies managed by CocoaPods** - If a Podfile is
+  present, we will run `pod install`, or `bundle exec pod install` if
+  both a Podfile and a Gemfile are present, and also cache the installed pods.
+* **Run the "test" build action for detected workspace (or project) and scheme
+  from the command line** - We will use `xctool` to build either the workspace
+  we find in your repo, or the project, if there is no workspace.
+
+
+### CocoaPods
+
+CircleCI will automatically detect if your project is using [CocoaPods](https://cocoapods.org)
+to manage dependencies. If you are using CocoaPods, then we recommend that you
+check your [Pods directory into source control](http://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control).
+This will ensure that you have a deterministic, reproducible build.
+
+If CircleCI finds a `Podfile` and the `Pods` directory is not present (or empty)
+ then we will run `pod install` to install the necessary dependencies in the
+`dependencies` step of your build.
+
+## Supported build and test tools
+
+CircleCI's automatic commands cover a lot of common test patterns, and you can customize your build
```

Comment:
No inference in 2.0, so anything that mentions automatic commands / automatic configuration should be removed